integration-tests/integration_tests/README.md

```diff
@@ -60,6 +60,183 @@ source ./component-test-source.sh
+      - MHS_DISABLE_SDS_TLS=True 
+      - MHS_SDS_REDIS_CACHE_HOST=redis 
+      - MHS_SDS_REDIS_DISABLE_TLS=True 
+  dynamodb: 
+    image: nhsdev/nia-dynamodb-local:1.0.1 
+    ports: 
+      - "8000:8000" 
+  rabbitmq: 
+    image: nhsdev/nia-rabbitmq-local:1.0.1  
+    ports: 
+      - "15672:15672" 
+      - "5672:5672" 
+    hostname: "localhost" 
+  redis: 
+    image: redis 
+    ports: 
+      - "6379:6379" 
+
+  inbound-lb: 
+    image: dockercloud/haproxy 
+    links: 
+      - inbound 
+    ports: 
+      - "443:443" 
+      - "8079:80" 
+    volumes: 
+      - /var/run/docker.sock:/var/run/docker.sock 
+    environment:  
+      - MODE=tcp 
+      - TIMEOUT=connect 0, client 0, server 0
+```
+
+2. Create a script to start the containers, call it export-env-vars-and-run-mhs-docker.sh and make it executable.
+
+3. Add the following content and your OpenTest details into the file
+
+```bash
+LIGHT_GREEN='\033[1;32m' 
+NC='\033[0m' 
+echo -e "${LIGHT_GREEN}Exporting environment variables${NC}" 
+
+export BUILD_TAG="latest" 
+export MHS_OUTBOUND_VALIDATE_CERTIFICATE="False"
+
+# Your OpenTest Party key here 
+export MHS_SECRET_PARTY_KEY="" 
+
+# Your OpenTest endpoint certificate here 
+export MHS_SECRET_CLIENT_CERT="" 
+
+# Your OpenTest endpoint private key here 
+export MHS_SECRET_CLIENT_KEY="" 
+
+# OpenTest Endpoint issuing subCA certificate and Root CA certificate here 
+export MHS_SECRET_CA_CERTS="" 
+
+echo -e "${LIGHT_GREEN}Stopping running containers${NC}" 
+docker-compose -f docker-compose.yml stop; 
+
+echo -e "${LIGHT_GREEN}Build and starting containers${NC}" 
+docker-compose -f docker-compose.yml up -d --build
+```
+
+4. Confirm you are connected to the OpenTest VPN and start all the containers by executing the shell script you created above.
+
+./export-env-vars-and-run-mhs-docker.sh
+
+#### Executing requests in Visual Studio Code
+1. Open Visual Studio Code
+2. Check out the integration-adapter project on the develop branch from github: https://github.com/nhsconnect/integration-adaptors
+3. In the project root create the directory `.vscode` if it doesn't exist already
+4. In the .vscode folder create a file called `settings.json`
+5. Add the information below to the `settings.json` file
+    ```json
+    {
+        "workbench.settings.editor": "json",
+        "workbench.settings.useSplitJSON": true,
+        "rest-client.environmentVariables": {
+            "$shared": {},
+            "$mhs": {
+                "ASID": "OpenTest ASID Here",
+                "GUID-idRoot": "3c996a6b-8e3b-42ec-8796-34aa8dca353c",
+                "GUID-idAltRoot": "af16f8c8-1a31-470c-802a-1716425b04fa",
+                "SHORT-TIMESTAMP": "20190927152035",
+                "PARTY-KEY": "OpenTest Party Key Here",
+                "FILE-UPLOADED": "test file will go here",
+                "LOCAL_SCR_URL": "http://localhost:9000/",
+                "MESSAGE_ID": "6a6c8179-e8e3-419a-a6dc-0c07844ddef7"
```

Comment:
I think it should be like this:
"BASE_URL": "http://localhost",
"INBOUND-PORT": "8082",
"OUTBOUND-PORT": "80",
"ROUTE-LOOKUP-PORT": "8088",
"FAKE-SPINE-PORT": "8091",
"ASID": "9XXXXXXXXXXX",
"PARTY-KEY": "A9XXXX-XXXXXXX",

GUID-idRoot, GUID-idAltRoot and MESSAGE_ID are generated with {{$guid}}.
LOCAL_SCR_URL is not used
SHORT-TIMESTAMP is predefined in requests
FILE-UPLOADED should not be in configuration, it is a String in one place of forward reliable request

Reply:
Surw, will add this in.